jupyterhub/spawner.py

```diff
@@ -230,7 +230,7 @@ def server(self, server):
                 self.orm_spawner.server = server.orm_server
         elif server is not None:
             self.log.warning(
-                "Setting Spawner.server for {self._log_name} with no underlying orm_spawner"
+                f"Setting Spawner.server for {self._log_name} with no underlying orm_spawner"
```

Comment:
the test fails with this error:
```py
      @property
      def _log_name(self):
          """Return username:servername or username
      
          Used in logging for consistency with named servers.
          """
          if self.name:
              return f'{self.user.name}:{self.name}'
          else:
  >           return self.user.name
  E           AttributeError: 'NoneType' object has no attribute 'name'
```

can someone give guidance of how to fix the test :)

Reply:
SHould be fixed by 6c2c5e5